cmake/cxxflags_common.cmake

```diff
@@ -14,18 +14,46 @@ endmacro()
 
 # Fixup default compiler settings
 if (MSVC)
-    add_compile_options(
+	# Addn'l -D #defines to make #include<windows.h> less painful
+	add_definitions(-DNOMINMAX=1 -DWIN32_LEAN_AND_MEAN=1)
+	add_compile_options(
         # Be as strict as reasonably possible, since we want to support consumers using strict warning levels
-        /W4 /WX
-        )
-else()
+        /W4 /WX)
+
+	# For some unknown reason, 'RelWithDebInfo' compiles with '/Ob1' as opposed to '/Ob2' which prevents inlining of
+	# functions not marked 'inline'. The reason we prefer 'RelWithDebInfo' over 'Release' is to get debug info, so manually
+	# revert to the desired (and default) inlining behavior as that exercises more interesting code paths
+	if ("${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo")
+		# TODO: This is currently blocked by an apparent Clang bug: https://github.com/llvm/llvm-project/issues/59690
+		# replace_cxx_flag("/Ob1" "/Ob2")
+		replace_cxx_flag("/Ob1" "/Ob2")
+	endif() # STREQUAL "RelWithDebInfo"
+
+	# Turn on "multi-procssor compilation" switch
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
```

Comment:
Added `/MP` here... So we use all the CPU cores when building... instead of "just one CPU core"